bundles/model/org.eclipse.smarthome.model.script/src/org/eclipse/smarthome/model/script/actions/ScriptExecution.java

```diff
@@ -91,6 +91,10 @@ public static Timer createTimer(AbstractInstant instant, Procedure0 closure) {
             dataMap.put("procedure", closure);
             dataMap.put("timer", timer);
             JobDetail job = newJob(TimerExecutionJob.class).withIdentity(jobKey).usingJobData(dataMap).build();
+            if (TimerImpl.scheduler.checkExists(job.getKey())) {
+                TimerImpl.scheduler.deleteJob(job.getKey());
```

Comment:
In order to get  shorter messages we could use the date time formatter and only display the last segment of the instance name.
Example: 
```groovy
        def instant = new DateTime()
        def closure = new Runnable() {public void run() {}}
        DateTimeFormatter dtf = DateTimeFormat.forPattern("MM/dd/yyyy HH:mm:ss")
        def nameArray = closure.toString().split("\\.")
        JobKey jobKey = new JobKey(dtf.print(instant) + ": " + nameArray[nameArray.length-1])
        println  jobKey
```
output: 
```
DEFAULT.12/21/2015 10:43:12: ScriptEngineOSGiTest$1@2eced48b
```